backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java

```diff
@@ -81,6 +84,7 @@ public void processPersonCSV(InputStream csvStream) throws IOException {
 
     private static CsvSchema personSchema() {
         return CsvSchema.builder()
+                .addColumn("facilityId", CsvSchema.ColumnType.STRING)
```

Comment:
A little weird to have just one column header that doesn't start upper case, no?

Reply:
we have a mix of header capitalization right now. This will need a much larger overhaul when we enable it through the UI